source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -55,6 +55,36 @@ void configSeparatorVisibility(ACRSeparator *view,
     view.isVisibilityObserved = YES;
 }
 
+ACRRtl getiOSRtl(std::optional<bool> const rtl)
```

Comment:
Perhaps use NSNumber (nil, YES, or NO), instead of custom enum?

Reply:
> It looks like we are manually computing the rtl property for each element based on all its ancestors. If you set `UISemanticContentAttributeForceRightToLeft` on a view, does the view not reverse all of its descendants?

sadly, no.

Reply:
NSNumber is there to wrap the enum value to store in NSArray. Enum is clear in conveying three distinct values the rtl property stands for.